addon/components/nrg-new-features/component.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { alias, bool } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import layout from './template';
+
+export default Component.extend({
+  layout,
+
+  whatsNewService: service('whats-new'),
+
+  isOpen: bool('content'),
```

Comment:
I think it would be better to have `bool('content')` as its own thing, and have `isOpen` have a default value. This would allow you set `isOpen` in `closeModal` instead of modifying `content`. It feels better at least to me.

```js
hasContent: bool('content'),
isOpen: reads('hasContent'),
```